### ISSUE TITLE
No scripting switch number not fully displayed

### ISSUE THREAD

**Issue by FTno** [uBlockOrigin/uBlock-issues#279]
### Prerequisites

- [ ] I verified that this is not a filter issue
    - Filter issues MUST be reported at [filter issue tracker](https://github.com/uBlockOrigin/uAssets/issues)
    
- [x] This is not a support issue or a question
    - Support issues and questions are handled at [/r/uBlockOrigin](https://old.reddit.com/r/uBlockOrigin/)
    
- [x] I performed a cursory search of the issue tracker to avoid opening a duplicate issue
    - Your issue may already be reported.
- I tried to reproduce the issue when...
    - [ ] uBlock Origin is the only extension
    - [ ] uBlock Origin with default lists/settings
    - [ ] using a new, unmodified browser profile
- [ ] I am running the latest version of uBlock Origin
- [ ] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I report is not a normal behavior

### Description

As seen in the screenshot the number at the far right is not displayed completely.

### A specific URL where the issue occurs

Here is a link to the page where I spotted this: https://www.facebook.com/brodogvin/timeline

### Steps to Reproduce

Enter a page there the number exceeds 2 digits

### Expected behavior:

[What you expected to happen]

### Actual behavior:

![screenshot](https://user-images.githubusercontent.com/16176811/47253898-6d423200-d45a-11e8-843b-c829932318b0.png)

### Your environment

* uBlock Origin version: 1.17.0
* Browser Name and version: Firefox 62
* Operating System and version: Win 7

**Comment by uBlock-user** [comment]
That's javascript switch and the number appears to get cut when it's more than 99.

**Comment by uBlock-user** [comment]
Setting the value of `margin-right` for span id `#no-scripting.hnSwitch fa` to `25px` fixes the issue.

**Comment by gwarser** [comment]
![image](https://user-images.githubusercontent.com/886325/50368620-0db3f000-058b-11e9-8a32-6866198f7e35.png)

**Comment by gorhill** [comment]
Is Noto Sans available by default on both Windows and MacOS?

**Comment by gwarser** [comment]
I don't think so: https://docs.microsoft.com/en-us/typography/fonts/windows_10_font_list

**Comment by gorhill** [comment]
I can reproduce with Firefox, I just forgot to test properly yesterday. Minding font face is not the right approach anyways. IWhat I am thinking now is that if the count is higher than 99, use a single symbol character to just convey that there is a whole lot of script tags found in the page. Now is to find which Unicode symbol would work for this.

**Comment by gorhill** [comment]
I thought about it but I fear it's going to be wrongly interpreted. I think I will just max out the count at 99, the exact number is not terribly important past low double-digit, it's just a quick estimate with not much decision-making value in being perfectly exact (which the code can't guarantee anyways given that the code can remove their own script tag from the DOM).

**Comment by uBlock-user** [comment]
> I thought about it but I fear it's going to be wrongly interpreted.

Why would it be wrongly interpreted ? 

Not showing the + sign when there's more than 99 scripts, misleads the person into thinking they're **exactly** 99 scripts on the page which is not the case on the aformendtioned URL by the OP.

**Comment by gorhill** [comment]
> misleads the person into thinking there are exactly 99 scripts

This will be documented that the values max out at 99. Nobody will make a decision based on whether the count is 99 or 121, it's all the same past low double digit. I sunk enough hours into that one trivial visual issue, I won't invest more time on this while there are things to fix which are far more important.

**Comment by uBlock-user** [comment]
But what's the issue in adding a `+` sign ? I don't understand that.

**Comment by gorhill** [comment]
It will be truncated. If not on your side, it will be on someone else's OS, browser, fonts, custom settings. The safest approach I see is to stick to 2 characters, this leaves enough space to make such issue again highly unlikely.

If I keep adding more spacing to accommodate safely all potential configs, then we end up with a large popup panel with more void areas just because of the per-site switches at the bottom. This also ends up pushing farther the dynamic filtering pane. etc I did not like the widened popup panel just for the sake of accommodating a count value which should be used as a _hint_ for decision-making, not as a logging tool -- that is the logger's purpose.

And related or not, now I am having to deal with more zoomed out popup panel on Android, making it more difficult to tap stuff unless you have delicate fingers, and I have no clue why this is happening, this is something I would like to fix before the next release.

**Comment by uBlock-user** [comment]
I see, thanks for the explanation.
